### PR TITLE
refactor New command to use ing/ and ins/ prefixes

### LINKED ISSUE #6: Add support for adding instructions
Add support for adding instructions to a recipe

### LINKED ISSUE #7: Add functionality to add ingredients to recipe
Sub-functions (to be done in v1.3):

- Ingredient prices
- Ingredient calories
- Ingredient quantity

### LINKED ISSUE #42: Add functionality to get ingredients and their quantities
Command to view the ingredients and their quantities from a given recipe

### PR DESCRIPTION
**Pull request by kevinswk94**
Also removed as many references to email and phone as possible

### CODE REVIEW

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 1]
```diff
@@ -37,13 +37,13 @@
...
+    private Label ingredients;
     @FXML
-    private Label phone1;
+    private Label ingredients1;
     @FXML
-    private Label email;
+    private Label instructions;
     @FXML
-    private Label email1;
+    private Label instructions1;
```
Comment (kevinswk94) [inline src/main/java/seedu/address/ui/PersonCard.java:46]: @AdarshChugani i updated these fields

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 2]
```diff
@@ -55,15 +55,15 @@ public PersonCard(Recipe recipe, int displayedIndex) {
 
         id.setText(displayedIndex + ". ");
         name.setText(recipe.getName().name);
-        phone.setText(recipe.getPhone().value);
-        email.setText(recipe.getEmail().value);
+        ingredients.setText(recipe.getIngredients().ingredientListString);
+        instructions.setText(recipe.getInstructions().instructionListString);
```
Comment (kevinswk94) [inline src/main/java/seedu/address/ui/PersonCard.java:59]: @AdarshChugani as well as the method called

File: src/main/java/seedu/address/ui/PersonCard.java (hunk 1 above)
Comment (AdarshChugani) [reply src/main/java/seedu/address/ui/PersonCard.java:46]: Looks good, thanks Kev! 👍

File: src/main/java/seedu/address/ui/PersonCard.java (hunk 2 above)
Comment (AdarshChugani) [reply src/main/java/seedu/address/ui/PersonCard.java:59]: 👍  Thanks Kev!

**Comment by AdarshChugani** [comment]
Thanks Kevin, Good job!

**Pull request merged**